README.md

```diff
@@ -64,35 +64,35 @@ The examples below use the version 5 syntax.
-* `hosts` - an array hosts Elasticsearch is running on.
-* `host` - the host Elasticsearch is running on.
-* `port` - the port Elasticsearch is running on.
-* `auth` - the authentication needed to reach Elasticsearch server. In the standard format of 'username:password'.
-* `protocol` - the protocol the Elasticsearch server uses. Defaults to http.
-* `hydrate` - whether or not to replace ES source by mongo document.
-* `filter` - the function used for filtered indexing.
-* `idsOnly` - whether or not returning only mongo ids in `esSearch`.
-* `countOnly` - whether or not returning only the count value in `esCount`.
-* `mappingSettings` - default settings to use with `esCreateMapping`.
-* `refreshDelay` - time in ms to wait after `esRefresh`. Defaults to 0.
-* `script` - whether or not the inline script are enabled in elasticsearch. Defaults to false.
-* `bulk` - options to use when synchronising.
-* `bulk.batch` - [batchSize](https://docs.mongodb.com/manual/reference/method/cursor.batchSize/) to use on synchronise options. Defaults to 50.
-* `bulk.size` - bulk element count to wait before calling `client.bulk` function. Defaults to 1000.
-* `bulk.delay` - idle time to wait before calling the `client.bulk` function. Defaults to 1000.
-
+- `index` - the index in Elasticsearch to use. Defaults to the collection name.
+- `type` - the type this model represents in Elasticsearch. Defaults to the model name. It may be a function `(modelName) => typeName`.
+- `client` - an existing Elasticsearch `Client` instance.
+- `hosts` - an array hosts Elasticsearch is running on.
+- `host` - the host Elasticsearch is running on.
+- `port` - the port Elasticsearch is running on.
+- `auth` - the authentication needed to reach Elasticsearch server. In the standard format of 'username:password'.
+- `protocol` - the protocol the Elasticsearch server uses. Defaults to http.
+- `hydrate` - whether or not to replace ES source by mongo document.
+- `filter` - the function used for filtered indexing.
+- `idsOnly` - whether or not returning only mongo ids in `esSearch`.
+- `countOnly` - whether or not returning only the count value in `esCount`.
+- `mappingSettings` - default settings to use with `esCreateMapping`.
+- `refreshDelay` - time in ms to wait after `esRefresh`. Defaults to 0.
+- `script` - whether or not the inline script are enabled in elasticsearch. Defaults to false.
+- `bulk` - options to use when synchronising.
+- `bulk.batch` - [batchSize](https://docs.mongodb.com/manual/reference/method/cursor.batchSize/) to use on synchronise options. Defaults to 50.
+- `bulk.size` - bulk element count to wait before calling `client.bulk` function. Defaults to 1000.
+- `bulk.delay` - idle time to wait before calling the `client.bulk` function. Defaults to 1000.
+- `onlyOnDemandIndexing` - whether or not to automatically index on CRUD operations. If set to false mexp middleware hooks for save, delete, update do not fire. Defaults to true.
```

Comment:
```diff
- whether or not to automatically index on CRUD operations. If set to false mexp middleware hooks for save, delete, update do not fire. Defaults to true.
+ whether or not to automatically index on CRUD operations. If set to true mexp middleware hooks for save, delete, update do not fire. Defaults to false.
```

`onlyOnDemandIndexing: true` - means that indexing should be called manually. Current implementation breaks my mind ;)) Please fix it.